draft-ietf-scitt-architecture.md

```diff
@@ -376,6 +376,23 @@ The SCITT architecture consists of a very loose federation of Transparency Servi
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Signed Inclusion Proofs.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
+SCITT consists of the following roles: Issuers, Transparency Services, Verifiers, and Auditors.
+The process works as follows:
+
+1. Issuers make Statements about Artifacts
+1. Issuers sign the Statements, producing Signed Statements
+1. Issuers submit the Signed Statements to Transparency Services
+1. Transparency Services take the submitted Signed Statements and insert them into an Append-only Log, producing Transparent Statements.
+This process can be thought of as Transparency Services counter-signing Issuers' Signed Statements
+1. Verifiers consume the Transparent Statements, and by verifying them, are convinced of the authenticity and integrity of Statements about an Artifact
+1. Auditors verify the correctness and consistency of the Transparent Statements
+
+Note that Issuers commit to a Statement by signing it and by submitting it to a Transparency Service.
+They are not necessarily the first ones nor the only ones who publish a Statement.
```

Comment:
```suggestion
They are not necessarily the first ones nor the only ones who register Signed Statements on one or more Transparency Services.
```

Reply:
Here I intentionally said "Statement" rather than "Signed Statement". If Alice and Bob sign the same statement, the Signed Statements will differ. But what I care about is the Statement that Alice and Bob agree on.

I rephrased this to clarify it.

Reply:
Thanks, @thgoebel. The nit I was looking to change was "publish" vs. "register". The process by which Signed Statements get added to the Append-only Log is through a registration policy. For consistency, we use "register" as opposed to publish, post, submit, etc. 

A verifier doesn't necessarily need to see agreement between various Issuers. A verifier can choose to trust, and filter in/out specific Issuers. An issuer may trust Alice and Bob, but statements made by Chuck are ignored. Whether they're malicious, or not relevant to the verifiers scenarios aren't overly important as there are scenarios where different issuers can make valid, yet conflicting statements based on a particular scenario or environment.

Reply:
this seems resloved. @SteveLasker could you please resolve, if it is?